TesseractCore/Importing.swift

```diff
@@ -0,0 +1,62 @@
+//  Copyright (c) 2015 Rob Rix. All rights reserved.
+
+private func nub<T: Equatable>(list: [T]) -> [T] {
+    var unique: [T] = []
+    for item in list {
+        if !contains(unique, item) {
+            unique.append(item)
+        }
+    }
+    return unique
+}
+
+public func parseEdge(edge: String) -> (String, String)? {
+    let characterParser = %("0"..."9") | %("a"..."z") | %("A"..."Z")
+    let ignoreBeginningOfLine = ignore("\t") ++ ignore("\"")
+    let edgeParser = ignoreBeginningOfLine ++ (characterParser+) ++ ignore("\" -> \"") ++ characterParser+ ++ ignore("\";");
+    
+    if let (rawSource, rawDestination) = edgeParser(edge)?.0 {
+        let source = String(rawSource.reduce([], +))
+        let destination = String(rawDestination.reduce([], +))
+        return (source, destination)
+    }
+    
+    return nil
+}
+
+public func importGraphViz(file: String) -> Graph<String>? {
```

Comment:
What do you think about `importDOT` and `exportDOT` instead?

I may end up adding support for [OGDL](http://ogdl.org) at some point as well. At that point I’m likely to consider abstracting serializations in general (but don’t sweat it in this PR :smile:).